x/interchainstaking/keeper/proposal_handler.go

```diff
@@ -239,32 +269,32 @@ func (k *Keeper) HandleUpdateZoneProposal(ctx sdk.Context, p *types.UpdateZonePr
 			k.SetZone(ctx, &zone)
 
 			// generate deposit account
-			portOwner := zone.ChainId + ".deposit"
+			portOwner := zone.ID() + ".deposit"
```

Comment:
```suggestion
			portOwner := fmt.Sprintf("%s.%s", zone.ID(), icstypes.ICASuffixDeposit)
```

or maybe even `zone.DepositPortName()` helper functions?

Reply:
Yeah, I have these helper functions being added in https://github.com/ingenuity-build/quicksilver/pull/420.  Not sure if we should duplicate the work.